.gitignore

```diff
@@ -36,3 +36,4 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+ .vscode/
```

Comment:
The `.vscode` folder will remain on the repository. Only that it can't be "accidentally" commited henceforth. Look [here](https://goo.gl/kZfdOD) to see how it can be removed entirely from the repo.